L/LibPQ/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "LibPQ"
+version = v"12.3"
+pg_version = string(version.major, '.', version.minor)
+tzdata_version = "2020a"
+
+# Collection of sources required to build LibPQ
+sources = [
+    ArchiveSource(
+        "https://ftp.postgresql.org/pub/source/v$pg_version/postgresql-$pg_version.tar.gz",
+        "708fd5b32a97577679d3c13824c633936f886a733fc55ab5a9240b615a105f50"
+    ),
+    ArchiveSource(
+        "https://data.iana.org/time-zones/releases/tzdata$tzdata_version.tar.gz",
+        "547161eca24d344e0b5f96aff6a76b454da295dc14ed4ca50c2355043fb899a2",
+    ),
+    ArchiveSource(
+        "https://data.iana.org/time-zones/releases/tzcode$tzdata_version.tar.gz",
+        "7d2af7120ee03df71fbca24031ccaf42404752e639196fe93c79a41b38a6d669",
+    ),
```

Comment:
I'd prefer to go in a separate JLL.  Maybe they can also be used by `TimeZones.jl`?  I didn't check what it needs.

Reply:
Yeah okay, that can happen. This isn't used at runtime at all but LibPQ already requires TimeZones so might as well have this.

Reply:
I take this is going to happen then? :slightly_smiling_face:

Reply:
I'm not really sure what to do here. 

These are only used at build time. I actually figured out a way to not use tzdata for this. So I just need tzcode for the build environment, not for any other. Do you still think it belongs in a JLL?

Reply:
Yes, if it's a built-time dependency it should be added as [`BuildDependency`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilderBase.BuildDependency).  This is particularly useful if it can be reused elsewhere, like for TimeZones, otherwise I guess when can quickly move here for the time being.

Reply:
TimeZones.jl doesn't use tzcode so there's no reuse possible there. And I can't find any other package which uses it.

Reply:
Ok ok, then let's go on with this version